deploy/helm/templates/rbac.yaml

```diff
@@ -57,6 +57,12 @@ rules:
       - get
       - create
       - update
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - delete
```

Comment:
This is required because we set Job the owner of a secret referred to by the Job spec.